### ISSUE TITLE
[Feature Request] Calendar Charms

### ISSUE THREAD

**Issue by Arty2** [marcoancona/MineTime#131]
Certainly just a minor detail, but it would be very nice to support charms, Outlook-like:

![image](https://user-images.githubusercontent.com/3519269/63984493-31451800-cad4-11e9-881c-cfdd63f49216.png)

Again congrats on the great work so far!

**Comment by marcoancona** [comment]
Yes, this has potential... may I know what is your use case of Outlook charms? I believe different people use it for different purposes.

**Comment by Arty2** [comment]
Our use case it handling several calendars (imported, different people etc), where charms give an additional optical way to differentiate between them.
A few examples: Plane charm for a travel related calendar, Ticket charm for an event calendar, Persons charm for meetings etc
